### PR TITLE
Setup mirroring for Android and iOS mirrors.

### PR DESCRIPTION
**Pull request by mitchdenny**
@weshaggard I've setup the ```-pr``` mirrors for Android and iOS, and from what I can tell this is the necessary configuration to start replicating the master branch. Can you confirm?

/cc @mayurid @bsiegel

### CODE REVIEW

**Comment by bsiegel** [comment]
FYI, Android also needs the `dev` branch mirrored, we need to get through an API review and migrate that to master soon though.

**Comment by mitchdenny** [comment]
I'm not sure if the mirroring logic can handle ```dev``` branch or not.

File: eng/pipelines/mirror-repos.yml [hunk 1]
```diff
@@ -53,6 +53,18 @@ jobs:
...
               azure-sdk/azure-sdk-for-python:
 
+          Azure/azure-sdk-for-ios:
+            Branch: master
+            TargetRepos: 
+              Azure/azure-sdk-for-ios-pr:
+              azure-sdk/azure-sdk-for-ios:
+
+          Azure/azure-sdk-for-android:
+            Branch: master
```
Comment (weshaggard) [inline eng/pipelines/mirror-repos.yml:63]: You can mirror the dev branch by changing master to dev here.

Comment (weshaggard) [reply eng/pipelines/mirror-repos.yml:63]: If we need both master and dev then we have to workaround a current limitation by adding the repo again below in the second template inclusion see python release/v3 branch below as an example.

**Comment by weshaggard** [comment]
triggered a test run https://dev.azure.com/azure-sdk/internal/_build/results?buildId=446404&view=logs&j=b5398758-6341-5a90-e082-88d59643ae40.

**Pull request merged**